nginx/common.sls

```diff
@@ -22,6 +22,11 @@
 /usr/share/nginx:
   file:
     - directory
+    
+apache2:
```

Comment:
Just tried it there and get Result True `The named service apache2 is not available`